kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala

```diff
@@ -0,0 +1,670 @@
+    def create: String = {
+      val partitions = fields.filter(_.isPartitionKey).map(_.name)
+      s"""
+         |CREATE TABLE ${table.qualifiedName} ${fields.map(_.toString).mkString("(", ", ", ")")}
+         |USING $format
+         |${if (partitions.nonEmpty) partitions.mkString("PARTITIONED BY (", ", ", ")") else ""}
+         |""".stripMargin
+    }
+  }
+
+  private val int: String = "INT"
+
+  private val char1 = "CHAR(1)"
+  private val char2 = "CHAR(2)"
+  private val char6 = "CHAR(6)"
+  private val char9 = "CHAR(9)"
+  private val char10 = "CHAR(10)"
+  private val char13 = "CHAR(13)"
+  private val char15 = "CHAR(15)"
+  private val char16 = "CHAR(16)"
+  private val char20 = "CHAR(20)"
+  private val char30 = "CHAR(30)"
+  private val char50 = "CHAR(50)"
+  private val char100 = "CHAR(100)"
+
+  private val varchar10 = "VARCHAR(10)"
+  private val varchar20 = "VARCHAR(20)"
+  private val varchar30 = "VARCHAR(30)"
+  private val varchar40 = "VARCHAR(40)"
+  private val varchar50 = "VARCHAR(50)"
+  private val varchar60 = "VARCHAR(60)"
+  private val varchar100 = "VARCHAR(100)"
+  private val varchar200 = "VARCHAR(200)"
+
+  private val dec_5_2 = "DECIMAL(5,2)"
+  private val dec_7_2 = "DECIMAL(7,2)"
+  private val dec_15_2 = "DECIMAL(15,2)"
+
+  private val date: String = "DATE"
+
+  val STORE_SALES: Table = {
+    val tableIdent = TableIdent("store_sales")
+    val fields = Seq(
+      Field("ss_sold_date_sk", int, isPartitionKey = true),
+      Field("ss_sold_time_sk", int),
+      Field("ss_item_sk", int),
+      Field("ss_customer_sk", int),
+      Field("ss_cdemo_sk", int),
+      Field("ss_hdemo_sk", int),
+      Field("ss_addr_sk", int),
+      Field("ss_store_sk", int),
+      Field("ss_promo_sk", int),
+      Field("ss_ticket_number", int),
+      Field("ss_quantity", int),
+      Field("ss_wholesale_cost", dec_7_2),
+      Field("ss_list_price", dec_7_2),
+      Field("ss_sales_price", dec_7_2),
+      Field("ss_ext_discount_amt", dec_7_2),
+      Field("ss_ext_sales_price", dec_7_2),
+      Field("ss_ext_wholesale_cost", dec_7_2),
+      Field("ss_ext_list_price", dec_7_2),
+      Field("ss_ext_tax", dec_7_2),
+      Field("ss_coupon_amt", dec_7_2),
+      Field("ss_net_paid", dec_7_2),
+      Field("ss_net_paid_inc_tax", dec_7_2),
+      Field("ss_net_profit", dec_7_2))
+    Table(tableIdent, fields)
+  }
+
+  val STORE_RETURNS: Table = {
+    val tableIdent = TableIdent("store_returns")
+    val fields = Seq(
+      Field("sr_returned_date_sk", int, isPartitionKey = true),
+      Field("sr_return_time_sk", int),
+      Field("sr_item_sk", int),
+      Field("sr_customer_sk", int),
+      Field("sr_cdemo_sk", int),
+      Field("sr_hdemo_sk", int),
+      Field("sr_addr_sk", int),
+      Field("sr_store_sk", int),
+      Field("sr_reason_sk", int),
+      Field("sr_ticket_number", int),
+      Field("sr_return_quantity", int),
+      Field("sr_return_amt", int),
```

Comment:
Spark use the `long` type to define these field type and some other, do we need to follow those ? Althought the standard tpcds seems does not require the type of `identifier` which allowed both `int` and `long`.

Reply:
FYI, https://github.com/apache/spark/pull/32037. I am going to change Spark's definition. It's no reason to use long for those fields

Reply:
ah I see, let's wait that PR merged first

Reply:
the spark pr merged, can you proceed?